services/apps/src/apps/routes/domain/permission.clj

```diff
@@ -0,0 +1,18 @@
+(ns apps.routes.domain.permission
+  (:use [common-swagger-api.schema :only [describe]]
+        [schema.core :only [defschema optional-key enum]])
+  (:import [java.util UUID]))
+
+(defschema AppIdList
+  {:apps (describe [String] "A List of app identifiers")})
+
+(defschema UserPermission
+  {:user       (describe String "The user ID")
+   :permission (describe String "The permission level assigned to the user")})
```

Comment:
We depend on those three being the only available things when calculating precedence anyway, looking lower, so I think it'd make sense to enforce it.